File: src/execution/ansi.zig
pub fn blue(comptime string: []const u8) []const u8 {
    return "\x1b[34m" ++ string ++ "\x1b[0m";
}

pub fn green(comptime string: []const u8) []const u8 {
    return "\x1b[32m" ++ string ++ "\x1b[0m";
}

pub fn lightGreen(comptime string: []const u8) []const u8 {
    return "\x1b[92m" ++ string ++ "\x1b[0m";
}

File: playground/src/ansi.js
export function blue(string) {
  return "\x1b[34m" + string + "\x1b[0m";
}

export function gray(string) {
  return "\x1b[90m" + string + "\x1b[0m";
}

export function green(string) {
  return "\x1b[32m" + string + "\x1b[0m";
}

export function underline(string) {
  return "\x1b[4m" + string + "\x1b[0m";
}
